### PR TITLE
Mock tests on CircleCI build

### PR DESCRIPTION
**Pull request by pkuczynski**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -2,10 +2,13 @@ machine:
...
+test:
+  override:
+    - "true"
+
 deployment:
-  develop:
+  production:
     branch: master
     commands:
-      - bundle install
```
Comment (pkuczynski) [inline circle.yml:9]: This is not needed, as it is run by default by CricleCI

File: circle.yml [hunk 2]
```diff
@@ -2,10 +2,13 @@ machine:
   ruby:
     version: "2.3.0"
 
+test:
+  override:
+    - "true"
+
 deployment:
-  develop:
+  production:
```
Comment (pkuczynski) [inline circle.yml:10]: We deploy to production, right? :)

File: circle.yml [hunk 3]
```diff
@@ -2,10 +2,13 @@ machine:
   ruby:
     version: "2.3.0"
 
+test:
+  override:
+    - "true"
```
Comment (pkuczynski) [inline circle.yml:7]: Prevents build from failing due lack of tests

File: circle.yml (hunk 2 above)
Comment (levino) [reply circle.yml:10]: Yes. does not get picked up by anything though I think.

**Review by levino — APPROVED** [review]

**Pull request merged**